include/neatvnc.h

```diff
@@ -49,6 +49,7 @@ typedef void (*nvnc_damage_fn)(struct pixman_region16* damage, void* userdata);
 typedef bool (*nvnc_auth_fn)(const char* username, const char* password,
                              void* userdata);
 typedef void (*nvnc_render_fn)(struct nvnc_display*, struct nvnc_fb*);
+typedef void (*nvnc_client_cut_fn)(struct nvnc_client*, char* data, uint32_t len);
```

Comment:
`data` should be `const`. The word "data" is such a generic word, would it not be more fitting to call this "text"? Why is there a length parameter? Can this not be a zero terminated string?

Reply:
Also, let's call the type `nvnc_cut_text_fn`.